styles.css

```diff
@@ -1,137 +1,146 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
 * {
   padding: 0;
   margin: 0;
   text-decoration: none;
   list-style: none;
   box-sizing: border-box;
-} 
+}
 
 /* Navigation */
+header {
+  background-color: #3c3a39;
+}
 
 nav {
   display: flex;
-  position: absolute;
   width: 100%;
-  height: 72px;
-  left: 0px;
-  top: 0px;
+  height: 10%;
   background-color: #3c3a39;
+  justify-content: space-between;
 }
 
 .logo-container {
-  position: absolute;
-  padding-top: 16px;
-  left: 142px;
-}
-#logo {
-  color: aliceblue;
-}
-
-.nav-links {
-  position: absolute;
-  left: 70%;
-  top: 16;
   display: flex;
-  flex-direction: row; 
   align-items: center;
+  text-align: center;
+  padding-top: 7.6%;
+  padding-left: 6%;
+  background-color: #3c3a39;
+  width: fit-content;
 }
 
-.navlinks {
-  padding: 7px;
-  color: aliceblue;
-  cursor:progress;
+#logo {
+  color: #fff;
+}
 
+.hamberger-btn {
+  text-align: center;
+  padding-top: 7.6%;
+  padding-left: 16%;
+  background-color: #3c3a39;
+  width: fit-content;
+  display: block;
 }
 
-.section {
- background-color: #030302;
- position: absolute;
- top: 72px;
- width: 100%;
- height: 798px;
+#ham {
+  padding: 6.5%;
+  position: absolute;
+  left: 83%;
+  top: 1%;
+  cursor: pointer;
 }
 
-.container{
-  position: relative;
-  background-repeat: no-repeat;
+.section {
+  background-color: #030302;
+  width: 100%;
+  height: 100%;
+  position: absolute;
   background-image: url(img/Illustration-header@2x.png);
-  background-position-y: 185px;
   background-attachment: fixed;
-  background-size: 40%;
+  background-size: 64%;
   background-position: right bottom;
+  background-position-y: 97%;
+  background-position-x: 179%;
+  padding-top: 10%;
   flex-grow: 1;
-  height: 100vh;
+  background-repeat: no-repeat;
 }
 
-.headline {
-  position: absolute;
-  left: 143px;
-  width: 566px;
-  height: 739px;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 192px 0px; 
-  gap: 24px;
-  
+.container {
+  height: 100%;
 }
 
 #myName {
-  width: 566px;
-  height: 160px;
-  color: #F55800;
-  font-family: 'Crete Round';
+  width: 100%;
+  color: #f55800;
+  font-family: "Crete Round", sans-serif;
   font-style: normal;
   font-weight: 400;
-  font-size: 72px;
-  line-height: 82px;
-
+  font-size: 350%;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  padding-top: 15%;
+  padding-left: 5%;
 }
```

Comment:
Please consider using the same `font-family and color properties etc` as used in the Figma template to meet the project requirements as you need to make an exact look-alike of the design.


```
font-family: 'Crete Round';
color: #FF6B00;
```